### DISCUSSION TITLE
nuxt可以js Gzip吗

### DISCUSSION THREAD

**Discussion by zhanmusiyuan** [nuxt/nuxt#9340] [Questions (Nuxt 2)]
nuxt可以js Gzip吗

**Comment by danielroe** [comment]
@zhanmusiyuan Would you provide more information? I'm not sure what your bug report or feature request is.

**Comment by akirarika** [comment]
@danielroe I'm Chinese, he should be saying: Nuxtjs can gzip the compiled `.js` file?

I actually suggest to deal with this problem in Nginx/Apache, Or use CDN (in China, [Upyun](https://www.upyun.com/) is not bad, it supports gzip or brotli adaptation according to the client).

**Reply by danielroe** [reply]
Thank you 😊

**Comment by danielroe** [comment]
For runtime compression (when using the Nuxt server), Nuxt uses [`compression` middleware](https://www.npmjs.com/package/compression), which you [can configure or swap out](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-render/#compressor).

For build-time gzipping you could use something like https://github.com/robcresswell/nuxt-compress. And of course the suggestion to handle this in your server or CDN is a good one too 👍
